homeassistant/components/elevenlabstts/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+  "domain": "elevenlabstts",
+  "name": "ElevenLabs Text-to-Speech (TTS)",
+  "codeowners": [],
```

Comment:
Do you not want to put yourself as a code owner?

Reply:
Done!